cve_bin_tool/checkers/wget.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+"""
+CVE checker for Wget
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-72/product_id-332/GNU-Wget.html
+"""
+import os
+from ..util import regex_find
+
+
+def get_version(lines, filename):
+    """
+    returns version information for wget found in given file.
+    VPkg: gnu, wget
+    """
+    regex = [r"wget: ([0-9]+\.[0-9]+\.[0-9]+)"]
+    version_info = dict()
+    if filename.startswith("wget"):
+        version_info["is_or_contains"] = "is"
```

Comment:
There's nothing to check if this "contains" wget, so your test-wget file below is never going to be detected.  Take a look at how other checkers have a guess_contains method.

Reply:
I looked into it, and have added a method for checking whether this contains wget. Thanks a lot for your response.

Reply:
Getting there!  This addition is good, but your file still won't be detected as wget.  Thsi time you may want to change your test file.